index.js

```diff
@@ -10891,7 +10971,15 @@ module.exports = {
                 on: {
                     ANSWER: [
                         {
-                            target: 'p-applicant-declaration'
+                            target: 'p-applicant-declaration',
+                            cond: [
+                                '==',
+                                '$.answers.p-applicant-who-are-you-applying-for.q-applicant-who-are-you-applying-for',
+                                'myself'
+                            ]
+                        },
+                        {
+                            target: 'p-main-applicant-declaration'
```

Comment:
Not important, but personally I'd flip this logic so that the PI journey is the default. Applicants applying for themselves is our most common type of journey, so I reckon should be the default.

Reply:
unless I'm misunderstanding something... that is what I have done. the first thing to be evaluated is the PI condition seeing how that is the most common route

Reply:
Sorry, by default, I meant it should be the 'fall through' condition - i.e. the target with no condition.

Reply:
what is the benefit of doing that as opposed to having the most commonly used route to be the first one that can be evaluated? that is what I meant by "misunderstanding something"